cdap-api/src/main/java/io/cdap/cdap/api/preview/DataTracer.java

```diff
@@ -40,4 +40,10 @@
    * Returns {@code true} if application is running in preview mode otherwise false is returned.
    */
   boolean isEnabled();
+
+  /**
+   * Returns number of records for the application
+   */
+  int getMaximumInputRecords();
```

Comment:
Overall good, could you rename the API? It's not clear why is it input. It traces all nodes. Name it somehow like getMaximumTracedRecords as soon as it's data tracer

Reply:
+1 `getMaximumTracedRecords` seems more clear!